third_party/terraform/utils/bootstrap_utils_test.go

```diff
@@ -51,10 +51,12 @@ func BootstrapKMSKeyWithPurpose(t *testing.T, purpose string) bootstrappedKMS {
 * a KMS key.
 **/
 func BootstrapKMSKeyWithPurposeInLocation(t *testing.T, purpose, locationID string) bootstrappedKMS {
```

Comment:
Other followup - I also want to rename these methods to have `Shared/Default/...` in the name so we know the name is the same across them, or add name param